Portfolio.html

```diff
@@ -0,0 +1,89 @@
+				</p>
+					<img src="https://y.yarn.co/ccf53d27-000e-4020-97e8-2a39bc93aed8_screenshot.jpg" 
+					alt="picture of me" 
+					class="aboutme_img">
+			</article>
+		</section>
+	
+		<section id="portfolio_container">
+			<h2 class="portfolio_title"><span class="portfolio_title_number">02.</span> Portfolio</h2>
+			<div class="project_container">
+				<h3 class="project_title">Project 1</h3>
+				<div class="project_content">
+					<img src="https://y.yarn.co/ccf53d27-000e-4020-97e8-2a39bc93aed8_screenshot.jpg" alt="project photo1" class="project_image">
+					<p class="project_text">Green vines attached to the trunk of the tree had wound themselves toward the top of the canopy. Ants used the vine as their private highway, avoiding all the creases and crags of the bark, to freely move at top speed from top to bottom or bottom to top depending on their current chore. At least this was the way it was supposed to be. Something had damaged the vine overnight halfway up the tree leaving a gap in the once pristine ant highway.</p>
+				</div>
+			</div>
+			<div class="project_container">
+				<h3 class="project_title">Project 2</h3>
+				<div class="project_content">
+					<img src="https://y.yarn.co/ccf53d27-000e-4020-97e8-2a39bc93aed8_screenshot.jpg" alt="project photo1" class="project_image">
+					<p class="project_text">Her eyebrows were a shade darker than her hair. They were thick and almost horizontal, emphasizing the depth of her eyes. She was rather handsome than beautiful. Her face was captivating by reason of a certain frankness of expression and a contradictory subtle play of features. Her manner was engaging.</p>
+				</div>
+			</div>
+			<div class="project_container">
+				<h3 class="project_title">Project 3</h3>
+				<div class="project_content">
+					<img src="https://y.yarn.co/ccf53d27-000e-4020-97e8-2a39bc93aed8_screenshot.jpg" alt="project photo1" class="project_image">
+					<p class="project_text">At that moment he had a thought that he'd never imagine he'd consider. "I could just cheat," he thought, "and that would solve the problem." He tried to move on from the thought but it was persistent. It didn't want to go away and, if he was honest with himself, he didn't want it to.</p>
+				</div>
+			</div>
+		</section>
+		<section id="git_container">
+			<h2 class="git_title"><span class="git_title_number">03.</span> Get in Touch </h2>
+			<div class="git_content">
+					<p class="git_text">I am currently looking for oppertunities and my inbox is always open. If it's a enquiry about collaborating or simply a question about anything on this site please don't hesitate to get in touch!</p>
+					<a href="#" class="git_button">Say Hi!</a>
+				</div>
+			</div>
+		</section>
+	
+		<div class="contact_container">
+			<!-- <h2 class="contact_title">Contact</h2> -->
```

Comment:
it is best not to have lines of commented out code in the repo - if it's not needed you can just delete it